docs/docs/job_registries.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "RQ: Job Registries"
+layout: docs
+---
+
+Each queue maintains a set of Job Registries:
+* `StartedJobRegistry` Holds currently executing jobs. Jobs are added right before they are 
+executed and removed right after completion (success or failure).
+* `FinishedJobRegistry` Jobs are added to this registry after they have been successfully completed.
+* `FailedJobRegistry` Jobs that have failed.
+* `DeferredJobRegistry` Deferred jobs (jobs that depend on another job and are waiting for that 
+job to finish).
+
+You can get the number of jobs in a registry, the ids of the jobs in the registry, and more. 
+Below is an example using a `StartedJobRegistry`.
+```python
+import time
+from redis import Redis
+from rq import Queue
+from rq.registry import StartedJobRegistry
+from somewhere import count_words_at_url
+
+conn = Redis()
+queue = Queue(connection=conn)
+job = queue.enqueue(count_words_at_url, 'http://nvie.com')
+
+# get StartedJobRegistry by queue
+registry = StartedJobRegistry(queue=queue)
+
+# or get StartedJobRegistry by connection
+registry = StartedJobRegistry(connection=conn)
```

Comment:
We should also add `name='queue_name'`. Without the `name` argument, it looks like `StartedJobRegistry` is an object that keeps track of all started jobs in this Redis connection.

Reply:
Got it. I've pushed a commit to address this.